docs/recipes.md

```diff
@@ -50,7 +50,7 @@ Headers can be added to your image requests in one of two ways. You can set head
 ```kotlin
 val request = ImageRequest.Builder(context)
     .data("https://www.example.com/image.jpg")
-    .setHeader("Cache-Control", "max-age=31536000")
```

Comment:
I think we should probably leave this as `max-age=31536000`. It's more typical to overwrite the `cache-control` header to cache for longer instead of disabling caching completely.